MOM6/patches/MOM_forcing_type.F90.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/MOM6/src/core/MOM_forcing_type.F90 b/MOM6/src/core/MOM_forcing_type.F90.new
+index 200bbd7..fca9187 100644
+--- a/MOM6/src/core/MOM_forcing_type.F90
++++ b/MOM6/src/core/MOM_forcing_type.F90.new
+@@ -2080,6 +2080,11 @@ subroutine fluxes_accumulate(flux_tmp, fluxes, G, wt2, forces)
+ 
+     fluxes%salt_flux(i,j) = wt1*fluxes%salt_flux(i,j) + wt2*flux_tmp%salt_flux(i,j)
+   enddo ; enddo
++  if (associated(fluxes%salt_flux_added) .and. associated(flux_tmp%salt_flux_added)) then
++    do j=js,je ; do i=is,ie
++      fluxes%salt_flux_added(i,j) = wt1*fluxes%salt_flux_added(i,j) + wt2*flux_tmp%salt_flux_added(i,j)
++    enddo ; enddo
++  endif
```

Comment:
Will this change the solutions for the non-bgc model?

Reply:
It shouldn't change any answers. `salt_flux_added` was previously only a diagnostic (and I _think_ it should be being accumulated anyway)